resources/default_pipeline_environment.yml

```diff
@@ -315,3 +315,13 @@ steps:
       developmentSystemId: null
   transportRequestUploadFile:
   transportRequestRelease:
+
+  #defaults for stage wrapper
+  piperStageWrapper:
+    extensionLocation: '.pipeline/extensions/'
+    globalExtensionLocation: '.pipeline/globalExtensions'
```

Comment:
Maybe not a point for this PR, but we are storing the extensions in the folder `pipeline`

Reply:
should the extensions be in a different place than the config?
Quite a while back we agreed to put the config for this lib into `.pipeline`, I guess this is then also the natural place for the extensions.
Anyway, this can be overwritten with custom-defaults as indicated here: https://sap.github.io/jenkins-library/configuration/